### DISCUSSION TITLE
Allow configuring S3 ACL

### DISCUSSION THREAD

**Discussion by rjbobeles** [directus/directus#5451] [Feature Requests]
1. ACL is properly assigned when uploading the original photo. However, when Directus uploads the cropped/shrinked photos, it returns an ACL error (see attached photo)

2. Upload a photo in directus
    Check bucket for images 

3. Directus: v9.0.0-rc62
    Mysql: MariaDB 10.5
    Deployment: Npx
    Browser: Brave
    
4. Additional Services: s3 (Linode Object Storage)
 
Sample Config:
```STORAGE_LOCATIONS="Linode,local"
STORAGE_LOCAL_DRIVER="local"
STORAGE_LOCAL_ROOT="./uploads"

STORAGE_LINODE_DRIVER="s3"
STORAGE_LINODE_ROOT="./directus/my-project"

STORAGE_LINODE_KEY="key"
STORAGE_LINODE_SECRET="secret"
STORAGE_LINODE_BUCKET="my-project"
STORAGE_LINODE_REGION=""
STORAGE_LINODE_ENDPOINT="ap-south-1.linodeobjects.com"
STORAGE_LINODE_ACL="public-read"
```

Images:
<img width="1440" alt="Screen Shot 2021-05-04 at 1 32 03 PM" src="https://user-images.githubusercontent.com/51547772/116964906-2dbd5780-acdf-11eb-8760-d8276a2a489b.png">
<img width="1440" alt="Screen Shot 2021-05-04 at 1 32 17 PM" src="https://user-images.githubusercontent.com/51547772/116964912-31e97500-acdf-11eb-89d8-98aa1deccbff.png">

**Comment by rijkvanzanten** [comment]
Ref https://github.com/directus/directus/discussions/4681

@rjbobeles What ACL value does Linode require? Directus doesn't set any at all at the moment

**Reply by rjbobeles** [reply]
Linode actually sets default values (see photo below) for the object ACLs. Based on my testing, not setting the .env will result in the object being set to private. In this scenario, the resized objects seem to be the ones acting up again.

[Link to Article](https://github.com/directus/directus/discussions/5451)
<img width="977" alt="Screen Shot 2021-05-04 at 9 41 29 PM" src="https://user-images.githubusercontent.com/51547772/117012480-92010b00-ad21-11eb-9dd8-989af1c1eb51.png">

**Reply by rijkvanzanten** [reply]
So just to confirm, regular files work as expected, but only the auto-generated thumbnails have the wrong ACL?

**Reply by rjbobeles** [reply]
Yes I that is what I got from using the s3 driver with different ACLs yesterday.

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
